js/bootstrap-datetimepicker.js

```diff
@@ -516,10 +516,6 @@
 			top = top - containerOffset.top;
 			left = left - containerOffset.left;
 
-            if( !elementOrParentIsFixed(this.element) ){
```

Comment:
Can that be the case, that it should be `if( elementOrParentIsFixed(this.element) )` (note missing exclamation mark) ?

I can imagine following logic : if some parent is fixed, calendar should "follow" scrolling, therefore we increase top by scroll position